sql/dapps.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var DappsSql = {
-	sortFields: ['type', 'name', 'category', 'link'],
+	sortFields: ['name'],
 
 	countByTransactionId: 'SELECT COUNT(*)::int AS "count" FROM dapps WHERE "transactionId" = ${id}',
```

Comment:
Please, review the unused functions and remove them. For example, I've checked `search` is obsoleted.
Also check the indentation in the file.